shard-row-id-bits.md

```diff
@@ -9,15 +9,27 @@ This document introduces the `SHARD_ROW_ID_BITS` table attribute, which is used
 
 ## Concept
 
-For the tables with a non-integer primary key or no primary key, TiDB uses an implicit auto-increment row ID. When a large number of `INSERT` operations are performed, the data is written into a single Region, causing a write hot spot.
+For the tables with a non-clustered primary key, no primary key or a multi-column primary key TiDB uses an implicit auto-increment row ID. When a large number of `INSERT` operations are performed, the data is written into a single Region, causing a write hot spot.
```

Comment:
```suggestion
For the tables with a non-clustered primary key or no primary key, TiDB uses an implicit auto-increment row ID. When a large number of `INSERT` operations are performed, the data is written into a single Region, causing a write hot spot.
```

Multi-column primary key can be clustered, which does not have an implicit row ID:

```sql
CREATE TABLE t (
    id INT,
    c1 INT,
    PRIMARY KEY (id, c1) CLUSTERED
) SHARD_ROW_ID_BITS = 4;

ERROR 8200 (HY000): Unsupported shard_row_id_bits for table with primary key as row id
```